mycore-restapi/src/main/java/org/mycore/restapi/v1/utils/MCRJSONFileVisitor.java

```diff
@@ -104,7 +104,7 @@ private void addBasicAttributes(Path path, BasicFileAttributes attrs) throws IOE
         if (attrs.isRegularFile()) {
             jw.name("contentType").value(MCRContentTypes.probeContentType(path));
             if (attrs instanceof MCRFileAttributes<?> fileAttrs) {
-                jw.name("md5").value(fileAttrs.md5sum());
+                jw.name("md5").value(fileAttrs.digest().getValue());
```

Comment:
I think its ok to deliver a sha512 hash as md5, because the api would be more stable for old applications. Alternative would be using the digest algo name as key.